modules/electrs.nix

```diff
@@ -48,7 +58,7 @@ in {
     systemd.services.electrs = {
       description = "Run electrs";
       wantedBy = [ "multi-user.target" ];
-      requires = [ "bitcoind.service" ];
+      requires = [ "bitcoind.service" "nginx.service"];
```

Comment:
I don't think `requires` is necessary here. electrs works fine without nginx.

Reply:
I see what you mean. The electrs systemd service doesn't need nginx, but SSL termination, which is essential to accessing electrs, only works if nginx is enabled.

Reply:
mmh, only for android afaik, but okay

Reply:
The way it's setup the Tor Hidden Service will direct traffic to the nginx reverse proxy which will only pass it on to electrs if the traffic uses SSL. So if nginx is disabled, electrs is unreachable, whether the user is on android or any other OS.

Reply:
ah, got it